source/_integrations/reolink.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Reolink NVR/camera
+description: Instructions on how to integrate Reolink devices (NVR/cameras) into Home Assistant.
+ha_category:
+  - Camera
+ha_iot_class: Local Polling
+ha_release: 2023.1
+ha_domain: reolink
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+ha_platforms:
+  - Camera
+ha_integration_type: integration
+---
+
+The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
+
+{% include integrations/config_flow.md %}
+
+## Reolink firmware limitations
```

Comment:
There is no need to create a paragraph and put two big warning boxes in it.
Just use normal text instead.

In general, known issues and limitations should be listed at the bottom of the documentation. These are not pre-requisites.

Reply:
done